### PR TITLE
Add MongoDB Parrot

### PR DESCRIPTION
**Pull request by OliverVermeulen**

### CODE REVIEW

**Comment by jmhobbs** [comment]
Did that do what you intended?  I don't see a stickbugparrot.gif in the files, only a mongodb logo parrot.

**Comment by OliverVermeulen** [comment]
Hello John, I removed the stickbugparrot.gif as it was to big and didn't
really fit with the tone of Party Parrot so I added the mongodbparrot.gif

Thank you
Oliver

File: parrots.yaml [hunk 1]
```diff
@@ -537,3 +537,5 @@
   name: Bad Bad Parrot
 - gif: dizzyparrot.gif
   name: Pixelated parrot dizzy after spinning
+- gif: mongodbparrot.gif
+  name: MongoDB Parrot
```
Comment (jmhobbs) [inline parrots.yaml:541]: Can you pop a newline at the end of the file here?  Thanks!

**Review by jmhobbs — CHANGES_REQUESTED** [review]

**Comment by jmhobbs** [comment]
Perhaps consider putting the logo behind the parrot in the style of these other logo parrots?

**Comment by jmhobbs** [comment]
Amazing! Thank you for your contribution!

**Pull request merged**